middleware/src/main/java/br/ufrn/dimap/middleware/infrastructure/lifecycleManagement/StaticRegistry.java

```diff
@@ -0,0 +1,29 @@
+package br.ufrn.dimap.middleware.infrastructure.lifecycleManagement;
+
+import br.ufrn.dimap.middleware.infrastructure.lifecycleManagement.modelo.IStaticRegistry;
+
+public class StaticRegistry implements IStaticRegistry

```

Comment:
Change interface name to StaticRegistryImpl, following the pattern used by the others classes and interfaces